# robot_modules_mode.msg
bool master_enable
bool yaw_enable
bool pitch_enable
bool chassis_enable
